.gitignore

```diff
@@ -1,8 +1,8 @@
 node_modules
 dist
 *.swp
-*.env
 vendor
 build
 .lambda
 .idea
+config/var_app
```

Comment:
`config/var_app` is only for the configurations to run the application locally. At this moment, we don't have any sensitive information in it, but maybe in the future, if the application updates, we might have. So I think ignoring it could help the developers run testing locally easier without encrypting the info.